bluepyopt/ephys/efeatures.py

```diff
@@ -22,6 +22,7 @@
 
 import logging
 
+from bluepyopt.ephys.serializer import DictMixin
```

Comment:
That's something we'd have to test.

I still want to write tests for all the other classes that serialize, to make sure I didn't miss anything.

I can do that in another PR though, if you're happy w/ this one.